public/js/admin/login.js

```diff
@@ -0,0 +1,61 @@
+// selecting elements from the html page
+const {
+  form, emailInput, passwordInput, loginBtn,
+} = querySelectors(
```

Comment:
@ahmedisam99 I know this isn't the focus of your PR, but I just had an idea for a future improvement of `querySelectors`.
If you used an array destructure instead of an object destructure, whenever you call it, you could avoid having to pass in the array of variable names as the first argument.  Instead, querySelectors would just return an array of the selected elements, and it would be a little simpler to call.  I imagine it would then look like this:
```
const [ //notice the array brackets now
  form, emailInput, passwordInput, loginBtn,
] = querySelectors(['.form', '.email', '.password', '.login'])  //now you don't need to tell the function what the names of your variables are!
```